modules/document_repository/README.md

```diff
@@ -0,0 +1,35 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents. Files may be uploaded and organized under any user-defined category.
```

Comment:
* Change: "non-candidate documents" to "documents relevant to the study across multiple candidates" 
* "user-defined category"  --> please add note below under Configurations describing how users can add a Category.